blockchain/types/transaction.go

```diff
@@ -551,7 +563,7 @@ func (tx *Transaction) WithSignature(signer Signer, sig []byte) (*Transaction, e
 	if err != nil {
 		return nil, err
 	}
-	cpy := &Transaction{data: tx.data}
+	cpy := NewTx(tx.data)
```

Comment:
cpy does not need to copy `tx.time`?

Reply:
It sets a time In `NewTx()`.

Reply:
I mean, the question was whether `cpy := NewTx(tx.data)` doesn't need to copy tx's time when copying tx.
I ask a question because it seems to work differently from the logic in Ethereum.
https://github.com/ethereum/go-ethereum/pull/21358/files#diff-8fb609018eccff10931645347515b8ecc194992cc4ef9e84ebdea87682dd50c3R257

Reply:
Oh, I misunderstand it. Thanks for review.